src/iconLists.ts

```diff
@@ -1,9 +1,19 @@
+export type baseIconKeys = typeof baseIconsList[number]
 export const componentIcons = ["RadioButton","RadioButtonLarge","Toggle","Preloader"]
+const componentIconsList = ["RadioButton","RadioButtonLarge","Toggle","Preloader"] as const
+export type componentIconKeys = typeof componentIconsList[number]
 export const doctypeIcons = ["Doc","GenericFile","Pdf","Ppt","Xls","Xml"]
+const doctypeIconsList = ["Doc","GenericFile","Pdf","Ppt","Xls","Xml"] as const
+export type doctypeIconKeys = typeof doctypeIconsList[number]
 export const illustrativeIcons = ["Authorisation","Book","Briefcase","BuildingAdministrative","Buildings","Catalog","ChartAnalytics","ChartPie","ChartScreen","ChartStatistic","ChatBubbles","Child","Cogwheel","Collaboration","Contract","Conversation","Court","CreditCards","Database","Mobile","Display","Doctor","Environment","Exchange","Failure","Family","Faq","Feedback","FileCabinet","Finance","Folder","Global","Group","Growth","HandCoins","HandPlate","Helpdesk","Home","House","IdBadge","LaptopContent","Laptop","Leap","Location","MagicWand","Mailbox","ManButtons","ManGlasses","ManLaptop","MessageSent","Messages","Meter","MigrationFinland","Money","MoneyBag","Organisation","MobileText","PiggyBank","Pillar","PlaneFlying","Presentation","Puzzle","Register","Rocket","ScaleBalance","Scale","Server","Shelter","Shop","Smartwatch","SocialSecurity","Steering","Success","Support","Swim","TabletText","Tablet","Team","Touch","Train","UserBadge","UserProfile","WebDevelopment","WebService","WomanButtons","WomanNecklace"]
+const illustrativeIconsList = ["Authorisation","Book","Briefcase","BuildingAdministrative","Buildings","Catalog","ChartAnalytics","ChartPie","ChartScreen","ChartStatistic","ChatBubbles","Child","Cogwheel","Collaboration","Contract","Conversation","Court","CreditCards","Database","Mobile","Display","Doctor","Environment","Exchange","Failure","Family","Faq","Feedback","FileCabinet","Finance","Folder","Global","Group","Growth","HandCoins","HandPlate","Helpdesk","Home","House","IdBadge","LaptopContent","Laptop","Leap","Location","MagicWand","Mailbox","ManButtons","ManGlasses","ManLaptop","MessageSent","Messages","Meter","MigrationFinland","Money","MoneyBag","Organisation","MobileText","PiggyBank","Pillar","PlaneFlying","Presentation","Puzzle","Register","Rocket","ScaleBalance","Scale","Server","Shelter","Shop","Smartwatch","SocialSecurity","Steering","Success","Support","Swim","TabletText","Tablet","Team","Touch","Train","UserBadge","UserProfile","WebDevelopment","WebService","WomanButtons","WomanNecklace"] as const
+export type illustrativeIconKeys = typeof illustrativeIconsList[number]
 export const logoIcons = ["LogoHorizontal","LogoHorizontalInvert","LogoVertical","LogoVerticalInvert"]
+const logoIconsList = ["LogoHorizontal","LogoHorizontalInvert","LogoVertical","LogoVerticalInvert"] as const
+export type logoIconKeys = typeof logoIconsList[number]
```

Comment:
This one as a example:
`*IconsList` is just a local variable to be used as `*Keys` as I did not want to change the use of existing `*Icons` string array because I do not know the use cases of it.

Reply:
Makes sense. The list was requested by users. Though I think most of the use cases are just iterating over the lists one way or another, so the "read only" typecast might be ok even for the original.